.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push]
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        rust: [stable]
+        include:
+          - os: ubuntu-latest
+            target: Linux
+          - os: macos-latest
+            target: Macos
+          - os: windows-latest
+            target: Windows
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Cargo audit
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cargo install --force cargo-audit
+          cargo generate-lockfile
+          cargo audit
+
+      - name: Cargo test
+        run: cargo test --verbose --features "content-blocking, resource-assembler"
```

Comment:
Is `cargo` installed by default on GitHub Actions runners? And is there a way to see the version being used?

Reply:
For cargo or for rust?

Reply:
ah I see there's a link in the run summary - https://github.com/actions/virtual-environments/blob/ubuntu20/20220410.2/images/linux/Ubuntu2004-Readme.md#rust-tools